openstack/compute/v2/servers/microversions.go

```diff
@@ -0,0 +1,14 @@
+package servers
+
+// Tags represents a servers tags.
+type Tags []string
+
+// ExtractTags will extract the tags of a server.
+// This requires the client to be set to microversion 2.26 or later.
+func (r serverResult) ExtractTags() (Tags, error) {
```

Comment:
If we return a `[]string`, do we need the `Tags` type?

Reply:
I felt it tied the field and return type together nicely. But given we return standard types everywhere else, I can change this.